internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -749,11 +755,22 @@ func parseWaitForFunctionArgs(
 	return js, popts, exportArgs(gargs), nil
 }
 
-func parseStringOrRegex(v sobek.Value) string {
+// Some getBy* APIs work with single quotes and some work with double quotes.
+// This inconsistency seems to stem from the injected code copied from
+// Playwright itself.
+//
+// I would prefer not to change the copied injected script code from Playwright
+// so that it is easier to copy over updates/fixes from Playwright when we need
+// to.
+func parseStringOrRegex(v sobek.Value, doubleQuote bool) string {
 	var a string
 	switch v.ExportType() {
-	case reflect.TypeOf(string("")):
-		a = fmt.Sprintf("'%s'", v.String()) // Strings require quotes
+	case reflect.TypeOf(string("")): // text values require quotes
```

Comment:
Should we have a constant here? Or do we expect the compiler to optimize?

Reply:
I didn't think about that tbh. I would expect the compiler to optimize it. I can make it a const though. I've added it to the catch all issue which I will work on after all the `getBy*` PRs have been merged.